src/dbnode/integration/commitlog_bootstrap_helpers.go

```diff
@@ -155,14 +155,8 @@ func writeCommitLogDataBase(
 		shardSet     = s.shardSet
 	)
 
-	// Write out commit log data
+	// Write out commit log data.
 	for currTs, blk := range data {
-		if specifiedTS != nil {
```

Comment:
Can you remove `specifiedTS` from the args to this method if it's not used/does anything any longer?

Reply:
I don't think I meant to delete this. Gonna put it back